File: python_ToSXs/python/06-thaphinh/20.py
height = 5

shape = ""

for i in range(1, height + 1):
    space = " " * (i - 1)

    character = [str(j) for j in range(1, height - i)]
    character = "".join(character)
    record = space + character
    shape += record + "\n"

print(shape)

File: cpp_e9zMZ/cpp/06-thaphinh/20.cpp
#include <iostream>

#include "zvn.h"
using namespace std;

int main() {
    int height = 5;
    string shape = "";

    for (int i = height; i >= 1; i--) {
        string record = myRepeat(" ", height - i);

        for (int j = 1; j <= i; j++) {
            record += to_string(j);
        }

        shape += record + "\n";
    }

    cout << shape << endl;
}